packages/next-swc/package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "scripts": {
     "build-native": "napi build --platform -p next-swc-napi --cargo-name next_swc_napi native --features plugin",
+    "build-native-no-plugin": "napi build --platform -p next-swc-napi --cargo-name next_swc_napi native",
```

Comment:
I think we need to add this script to our turbo config here https://github.com/vercel/next.js/blob/d5706dc4175051528e2cf43132c4ae7ad447abe6/turbo.json#L3